scripts/vscode.patch

```diff
@@ -824,6 +824,16 @@ index 74fc798..03728d0 100644
 +			// 	supportFetchAPI: true,
 +			// 	corsEnabled: true
 +			// });
+@@ -310 +328 @@
+-
```

Comment:
We might want to remove all the extra whitespace here so the patch is as small as possible.